src/LongRunning/Gapic/OperationsGapicClient.php

```diff
@@ -48,8 +48,10 @@
 use Google\GAX\AgentHeaderDescriptor;
 use Google\GAX\ApiCallable;
 use Google\GAX\CallSettings;
+use Google\GAX\GrpcConstants;
 use Google\GAX\GrpcCredentialsHelper;
 use Google\GAX\PageStreamingDescriptor;
+use Google\GAX\PathTemplate;
```

Comment:
This is also not used in this file.

Reply:
fixing cs php-cs-fixer in artman fixed the problem ([artman PR](https://github.com/googleapis/artman/pull/310))